src/utils/copy.rs

```diff
@@ -94,7 +67,7 @@ mod test {
         let destination = tempdir::TempDir::new("cratesfyi").unwrap();
 
         // lets try to copy a src directory to tempdir
-        let res = copy_dir(Path::new("src"), destination.path());
+        let res = copy_doc_dir(Path::new("src"), destination.path());
```

Comment:
Since this test deals with `copy_doc_dir` now, the test name should also reflect this --> `test_copy_doc_dir`

Also this test is `#[ignore]` which is a bit unfortunate, it would be nice to check if we can get this test working again (or delete it otherwise). --> also for another PR?